src/ClientWidgetApi.ts

```diff
@@ -471,13 +471,17 @@ export class ClientWidgetApi extends EventEmitter {
             await this.transport.reply<IWidgetApiErrorResponseData>(request, {
                 error: {message: "Invalid request - missing event contents"},
             });
+        } else if (typeof request.data.encrypt !== "boolean") {
+            await this.transport.reply<IWidgetApiErrorResponseData>(request, {
+                error: {message: "Invalid request - missing encryption flag"},
+            });
```

Comment:
Does this need to handle backward compatibility?

Reply:
No, there's nothing actually using these APIs yet, and the MSC is still subject to change